pkg/reconciler/revision/table_test.go

```diff
@@ -309,6 +311,28 @@ func TestReconcile(t *testing.T) {
 				MarkInactive("NoTraffic", "This thing is inactive."), withObservedGeneration(1)),
 		}},
 		Key: "foo/pa-inactive",
+	}, {
+		Name: "pa inactive II",
```

Comment:
can we call this something that says how it's different? e.g. `pa inactive, and initial scale target never reached`